utils/meta_infos.json

```diff
@@ -56,7 +68,12 @@
     "legalnero": "LNR",
     "lener_br": "LNB",
     "mapa": "MAP",
-    "turkish_constitutional_court_decisions_judgment": "TCD"
+    "turkish_constitutional_court_decisions_judgment": "TCD",
+    "criticality_prediction": "CP",
```

Comment:
should this be swiss_criticality_prediction: SCP?

Reply:
And the sub datasets should be removed here, right?

Reply:
I think line 72 is unnecessary.